trino/auth.py

```diff
@@ -18,6 +18,7 @@
 
 from six import with_metaclass
 from typing import Any, Optional, Text  # NOQA
+from requests.auth import AuthBase
```

Comment:
That's out of scope of this PR, but we should define an __ALL__ here, so that `from trino.auth import *` doesn't pull things like `AuthBase`... or we say we don't care.